js/samples/08.datasource.azureAISearch/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "teams-azureaisearch-demo",
+    "version": "1.0.0",
+    "description": "Sample bot demonstrating using Azure AI Search for RAG.",
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./lib/index.js",
+    "engines": {
+        "node": ">=18.0.0"
+    },
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .env npm run dev",
+        "dev:teamsfx:testtool": "env-cmd --silent -f .env npm run dev",
+        "dev:teamsfx:launch-testtool": "env-cmd --silent -f env/.env.testtool teamsapptester start",
+        "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
+        "build": "tsc --build",
+        "clean": "rimraf node_modules lib",
+        "lint": "eslint **/src/**/*.{j,t}s{,x} --fix --no-error-on-unmatched-pattern",
+        "start": "tsc --build && node ./lib/index.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "watch": "nodemon --watch ./src -e ts --exec \"yarn start\"",
+        "indexer:create": "yarn build && env-cmd --silent -f .env node ./lib/indexer/setup.js",
```

Comment:
should be:
`"indexer:create": "yarn build && env-cmd --silent -f .env node ./lib/indexers/setup.js"`